<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>332</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Meiryo UI</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Optical Chopper</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>301</width>
     <height>281</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="graphical_tab">
    <attribute name="title">
     <string>Graphical</string>
    </attribute>
    <widget class="QPushButton" name="on_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Meiryo UI</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 200, 0);
font: bold 14pt &quot;Meiryo UI&quot;;</string>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="off_button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Meiryo UI</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(250, 0, 0);
font: bold 14pt &quot;Meiryo UI&quot;;</string>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="freq_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Meiryo UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>7</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="hz_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Meiryo UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hz</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="console_tab">
    <attribute name="title">
     <string>Console</string>
    </attribute>
    <widget class="QPlainTextEdit" name="read_area">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>311</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="console_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="controller_tab">
    <attribute name="title">
     <string>Controller</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
